peekaboo/toolbox/known.py

```diff
@@ -73,6 +73,11 @@ def last_result(self):
             return self.sample_journal[-1].result
         return Result.unknown
 
+    @property
+    def result(self):
```

Comment:
Why is this property added here if the other hunk of the commit isn't using it?

Reply:
Sorry to take this up again: The documentation of the `result` report properties and example rules using it are added two commits prior. So the addition of the property should go into that patch as well to have it self-contained and in a working state.

Reply:
After some more review and understanding your intent here, I've moved this into the patch where its surrounding logic is added. The same was necessary for db.analysis_journal_fetch_journal. I've added a test to the testsuite for it as well. In the final patch I dropped the now redundant db.sample_info_fetch code and tests altogether.